pkg/rtc/types/interfaces.go

```diff
@@ -80,6 +80,7 @@ const (
 	ParticipantCloseReasonSimulateMigration
 	ParticipantCloseReasonSimulateNodeFailure
 	ParticipantCloseReasonSimulateServerLeave
+	ParticipantCloseReasonNegotiateFailed
```

Comment:
Can you add this to the `String()` method below also?

Reply:
we also need to add this to ToDisconnectReason: it should be `DisconnectReason_STATE_MISMATCH`